src/libultra/io/pfsinitpak.c

```diff
@@ -1,5 +1,126 @@
+#include "PR/pfs.h"
 #include "global.h"
 
-#pragma GLOBAL_ASM("asm/non_matchings/boot/pfsinitpak/osPfsInitPak.s")
+s32 __osPfsCheckRamArea(OSPfs* pfs);
 
-#pragma GLOBAL_ASM("asm/non_matchings/boot/pfsinitpak/__osPfsCheckRamArea.s")
+s32 osPfsInitPak(OSMesgQueue* queue, OSPfs* pfs, s32 channel) {
+    s32 ret;
+    u16 sum;
+    u16 isum;
+    u8 temp[BLOCKSIZE];
```

Comment:
Needs updating elsewhere too
```suggestion
    u8 buf[BLOCKSIZE];
```